addons/a11y/src/components/Report/Rules.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { FunctionComponent } from 'react';
 import { styled } from '@storybook/theming';
-
-import { Icons } from '@storybook/components';
+import { Badge, Icons } from '@storybook/components';
 import { CheckResult } from 'axe-core';
+import { RuleType } from '../A11YPanel';
+import { SizeMe } from 'react-sizeme';
```

Comment:
We could cut back on the number of dependencies by using https://github.com/storybooks/storybook/blob/tech/loader-scope-changes/lib/ui/src/app.js#L4

```js
import ResizeDetector from 'react-resize-detector';
```

Reply:
That's a good point. I hadn't spotted that in the repo. It looks like it also has the "refreshMode" feature so it fits the needs. Ill convert what I have to use react-resize-detector. Thanks!

Reply:
So I tried the react-resize-detector and there is a delay between when you start an action and the resizing occurs. Think We should switch back to ResizeMe where this did not occur. react-sizeme does an initial, invisible render of the component to measure its size. Then, when the component renders, the size is known and the component can render properly from the beginning.
![glitchy-response](https://user-images.githubusercontent.com/29763590/56427077-85b7cb80-6288-11e9-8214-3ade25af156b.gif)

Reply:
I tried switching out the react-resize-detector for resize me but for that use case, react-resize-detector was the better library

Reply:
@ndelangen what do you think about keeping both?